packages/crew_profile.rb

```diff
@@ -0,0 +1,48 @@
+require 'package'
+
+class Crew_profile < Package
+  description 'A profile for Chromebrew, to be used instead of ~/.*rc'
+  homepage 'https://github.com/skycocker/chromebrew'
+  version '0.4.3'
```

Comment:
How did you come up with this version?  Seems like 1.0 would make more sense?

Reply:
No. Since it is `A profile for Chromebrew`, I figured it should be correlated to the Chromebrew version.

Reply:
It was a while ago.